tests/distributions/test_rejector.py

```diff
@@ -61,9 +61,8 @@ def test_rejector(rate, factor):
 @pytest.mark.parametrize('factor', [0.25, 0.5, 1.0])
 def test_exponential_elbo(rate, factor):
     num_samples = 100000
-    rates = torch.tensor(rate).expand(num_samples, 1).requires_grad_(True)
-    factors = torch.tensor(factor).expand(num_samples, 1).requires_grad_(True)
-
+    rates = torch.tensor(rate).expand(num_samples, 1)
```

Comment:
How about `torch.Tensor(num_samples, 1).fill_(rate).requires_grad_()` for tests that require grad?